app/code/Magento/InventoryInStorePickup/etc/config.xml

```diff
@@ -22,5 +22,17 @@
                 <name>Store pickup</name>
             </ident_storepickup>
         </trans_email>
+        <carriers>
+            <in_store>
+                <active>0</active>
+                <sallowspecific>0</sallowspecific>
+                <model>Magento\InventoryInStorePickup\Model\Carrier\InStorePickup</model>
+                <name>Pick in Store</name>
+                <title>In-Store Delivery</title>
```

Comment:
Delivery?

Reply:
In-Store Pickup Delivery method with Pick in Store rate.
I will be happy to see other variants :)